src/solver/batchnorm/backward_spatial_single.cpp

```diff
@@ -131,7 +131,7 @@ BnBwdTrainingSpatialSingle::GetSolution(const ExecutionContext& context,
         else if(in_nhw < (32 * 1024 * 1024) && in_cstride > 512)
         {
             variant    = (n >= 32) ? 1 : 3;
-            xlocalsize = std::min(64 * ((in_cstride + 63) / 64), static_cast<unsigned int>(1024));
```

Comment:
What is 64 in the old code? Is it connected to the WAVESIZE (which is 32 for Navi21)? Ditto all other places where literal 64 is used.

Reply:
yes Artem.  It is related to WAVESIZE (64).

Reply:
Ok. This change looks good for urgent fixing of 5.0. But shall we keep this machinery in a full-blown fix, but switching 64 to 32 for Navi21?